BeatLeader.csproj

```diff
@@ -144,6 +144,9 @@
       <Private>False</Private>
       <SpecificVersion>False</SpecificVersion>
     </Reference>
+    <Reference Include="VRUI, Version=0.0.0.0, Culture=neutral, PublicKeyToken=null">
+      <HintPath>..\..\..\..\Games\Steam\steamapps\common\Beat Saber v1.21.0\Beat Saber_Data\Managed\VRUI.dll</HintPath>
```

Comment:
Should be `<HintPath>$(BeatSaberDir)\Beat Saber_Data\Managed\VRUI.dll</HintPath>`